api/tests/core/offerers/test_api.py

```diff
@@ -656,7 +656,7 @@ def test_max_limit_number_of_venues(self) -> None:
         with assert_num_queries(1):
             venues = list(offerers_api.get_eligible_for_search_venues(max_venues=1))
 
-        assert venues[0].id == eligible_venues[0].id
+        assert {venues[0].id} <= {venue.id for venue in eligible_venues}
```

Comment:
(détails / style / subjectif) un peu plus simple, peut-être : 

```suggestion
        assert venues[0].id in {venue.id for venue in eligible_venues}
```

(instant confession : je n'ai jamais utilisé l'opérateur `<=` sur un set)